frontend/app/src/components/StreamlitDialog/ThemeCreatorDialog.test.tsx

```diff
@@ -275,6 +267,6 @@ describe("Opened ThemeCreatorDialog", () => {
     expect(navigator.clipboard.writeText).toHaveBeenCalledWith(`[theme]
 base="light"
 `)
-    expect(updateCopied).toHaveBeenCalledWith(true)
+    expect(screen.getByText("Copied to clipboard")).toBeInTheDocument()
```

Comment:
is this the text we see when clicking and that goes away after some time? Could we run into a potential flakiness here if the check happens after the text has already vanished?

Reply:
Good callout. I don't see any timers for it to disappear in this case explicitly.